.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -5,14 +5,14 @@ if ($Env:RUNNER_OS -eq 'Windows')
 else
 {
     $dockerRunSwitches = @(
-        '--name', 'sql2019'
+        '--name', 'sql2022'
```

Comment:
DRY, please use a variable or script parameter to store the SQLServer version. Do it here and below.